src/CMakeLists.txt

```diff
@@ -317,6 +343,10 @@ else()
 			set(CLIENT_PLATFORM_LIBS ${CLIENT_PLATFORM_LIBS} ${XXF86VM_LIBRARY})
 		endif(NOT HAIKU AND NOT APPLE)
 
+		if(ENABLE_TOUCH)
+			find_library(XINPUT_LIBRARY Xi REQUIRED)
+		endif()
```

Comment:
One more small suggestion:
Add Xi to CLIENT_PLATFORM_LIBS here so you don't have to explicitly mention it below.

Reply:
Done!